frontend/app-development/features/dataModelling/DataModelling.tsx

```diff
@@ -14,9 +17,11 @@ export function DataModelling({
   createPathOption = false,
 }: DataModellingProps): ReactNode {
   const { t } = useTranslation();
-  const { status, error, data } = useDatamodelsMetadataQuery();
+  const { org, app } = useParams<{ org: string; app: string }>();
```

Comment:
Dette forslaget er litt utenfor scope til denne PR, men burde vi laget en type for URLParams? Jeg mistenker at vi bruker useParams for å hente ut org og app mange steder i Studio? 🤔 

```suggestion
  const { org, app } = useParams<URLParams>();
```

Reply:
Helt enig, kan lage en egen PR for det!